flake.nix

```diff
@@ -3,7 +3,7 @@
     common.url = "github:nammayatri/common";
 
     # Backend inputs
-    shared-kernel.url = "github:nammayatri/shared-kernel";
+    shared-kernel.url = "github:nammayatri/shared-kernel/382fac5fd91cbc6b928f050285cfeed12177c5c2";
```

Comment:
This should be reverted. See https://github.com/nammayatri/nammayatri/tree/main/Backend#updating-flake-inputs

Reply:
Yup, this file was pushed by mistake, removed